client/js/app.modules/main/main.controller.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
I suppose "Main" is to generic name. Mb use some more specific name like "Navigation" ,"Header", "Menu".
The same for controllers/services